src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/JavadocTagContinuationIndentationCheck.java

```diff
@@ -58,6 +68,32 @@
  *   &lt;property name="offset" value="2"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <pre>
+ * &#47;**
+ *  * @tag comment..
+ *  * Indentation spacing is 0 (violation)
+ *  *   Indentation spacing is 2 (OK)
+ *  *  Indentation spacing is 1 (violation)
+ *  *&#47;
+ * public class Test {
+ * }
+ * </pre>
+ * <p>
+ * To configure the check to show violations for Tight-HTML Rules:
+ * </p>
+ * <pre>
+ * &lt;module name="JavadocTagContinuationIndentation"&gt;
+ *   &lt;property name="violateExecutionOnNonTightHtml" value="true"/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <pre>
+ * &#47;**
+ *  * &lt;p&gt;HTML tag is unclosed (violation)
+ *  * &lt;p&gt;HTML tag is closed&lt;/p&gt; (OK)
```

Comment:
`HTML tag is closed`
please use some valid html tag, and add explanation to violation/OK comment

Reply:
done